python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -149,6 +154,7 @@ def post(self):
         if lat and lon:
             mine.mine_location = MineLocation(latitude=lat, longitude=lon)
             cache.delete(MINE_MAP_CACHE)
+            self.call_rebuild_map_cache()
```

Comment:
I'd recommend spawning the thread after the DB object has been saved/committed or the return statement to ensure the new cached values already exist in the DB.

Probably can throw in another conditonal there to look for all the fields that are displayed on the mine pin like Ryan suggested earlier

Reply:
Good catch about the commi/cache order.  Though putting a check in the post shouldn't be needed since any created mine will have a location. I will write one for the put method